pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -339,6 +341,20 @@ func (v *vpnSeedServer) Deploy(ctx context.Context) error {
 									Value: v.podNetwork,
 								},
 							},
+							ReadinessProbe: &corev1.Probe{
+								Handler: corev1.Handler{
+									TCPSocket: &corev1.TCPSocketAction{
+										Port: intstr.FromInt(openVPNPort),
+									},
+								},
+							},
+							LivenessProbe: &corev1.Probe{
```

Comment:
It would be nice to have also some initial delay and probably to use custom timeout instead of the default of 1s.
Once the liveness probe fail, kubelet will restart the container. To avoid restarts on sporadic failures, you can set `failureThreshold` to something greater than the default of `1`, which will restart immediately after each failed check.

Reply:
@vpnachev From https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes it looks like the `failureThreshold` default value is `3`, which seems reasonable for these probes.
What makes you think that an initial delay and a custom timeout are required here?

Reply:
Ok, I have overlooked the default for `failureThreshold`. The default looks already good enough.
But for the initial delay and timeout, I have not much experience with those containers, but are they able to start-up and be ready in less than 1 second?

Reply:
Let's give it a try with the default values.